jscomp/ext/ext_path.ml

```diff
@@ -322,3 +322,32 @@ let find_package_json_dir cwd  =
   find_root_filename ~cwd  Literals.bsconfig_json
 
 let package_dir = lazy (find_package_json_dir (Lazy.force cwd))
+
+let getchdir s =
+  let p = Sys.getcwd () in
+  Unix.chdir s;
+  p
+
+let normalize s =
+  try getchdir (getchdir s) with _ -> s
+
+let real_path p =
+  match (try Some (Sys.is_directory p) with Sys_error _ -> None) with
```

Comment:
Note `ext` does not depend on Unix, we can depend on Unix in bsb, we can create a file bsb_real_path in bsb directory

Reply:
done